example.csharp/addons/tiltfive/scenes/T5XRRig.cs

```diff
@@ -0,0 +1,47 @@
+using Godot;
+using System;
+
+public partial class T5XRRig : SubViewport
+{
+	T5OriginCS origin;
+	T5CameraCS camera;
+	T5ControllerCS[] wands = new T5ControllerCS[4];
+
+	public string GlassesID { get; set; } 
+	public T5Interface.GameboardType GameboardType { get; set; }
+	public Aabb GameboardSize { get; set; }
+	public T5OriginCS Origin {  get { return origin; } }
+	public T5CameraCS Camera{ get { return camera; } }
+	public T5ControllerCS Wand(int i) 
+	{
+		if(i >= 0 && i < wands.Length) { return wands[i]; }
+		return null;
+	}
+
+	static string[] wandPaths = { "Origin/Wand_1", "Origin/Wand_2", "Origin/Wand_3", "Origin/Wand_4" };
+
+	// Called when the node enters the scene tree for the first time.
+	public override void _Ready()
```

Comment:
Finding the Origin/Camera/Wand nodes should be done in _EnterTree() rather than _Ready() so child nodes will be able to query for them in their _Ready() functions.